### DISCUSSION TITLE
Connect Node (NO hierarchy)

### DISCUSSION THREAD

**Discussion by pardonmycode** [freeplane/freeplane#1712] [Ideas]
Sometimes I wanna connect Nodes in an non hierarchy way. 
For example: Graph theory, UML diagrams etc.
How you think about it?

**Comment by euu2021** [comment]
You can use Connectors (`Insert->Connect`)

Take a look at this example: https://sourceforge.net/p/freeplane/discussion/758437/thread/5c60cc647a/
